manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
+	gnome-menus \
```

Comment:
What does this do?

Reply:
Gnome keyring allows for password storage in the desktop. Gnome Menus is used on the "Classic" flavor of Gnome. But can be do without.

Reply:
Let's skip gnome-menus then.

Reply:
Dropped